man/systemd-homed-sync-bulk.service.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+
+<!--
+  SPDX-License-Identifier: LGPL-2.1-or-later
+
+  Copyright © 2023 GNOME Foundation Inc.
+        Original Author: Adrian Vovk
+-->
+
+<refentry id="systemd-homed-sync-bulk.service" conditional='ENABLE_HOMED'
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>systemd-homed-sync-bulk.service</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>systemd-homed-sync-bulk.service</refentrytitle>
+    <manvolnum>8</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>systemd-homed-bulk-sync.service</refname>
+    <refpurpose>Home Area Bulk Directory Sync</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>systemd-homed-bulk-sync.service</filename></para>
+    <para><filename>systemd-homed-bulk-sync.path</filename></para>
+    <para><filename>/usr/lib/systemd/systemd-homed-sync-bulk</filename></para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>systemd-homed-sync-bulk</command> is a user service that synchronizes a user's
+    bulk directory between two locations: one in the user's home directory, and another belonging
+    to the system. It is run at login and whenever the contents of either directory change.</para>
+
+    <para>The bulk directory is for storing data that conceptually belongs to the user's identity (i.e. their
+    <ulink url="https://systemd.io/USER_RECORD/">JSON User Record</ulink>), but is too large to embed directly
+    into the JSON. For example: the bulk directory should be used to store a custom profile picture for the user.
+    Another example: the bulk directory could be used to store the user's wallpaper, which will then allow the
+    system's login UI to display it.</para>
+
+    <para>As mentioned above, there are two copies of the bulk directory that are kept in sync. The first is
+    <filename>$XDG_DATA_HOME/homed/</filename> or <filename>$HOME/.local/share/homed/</filename>, which we will
+    call the "home bulk" directory. The home bulk directory is only accessible while the user is in an active
+    state (i.e. logged in and unlocked). The other copy is found in
+    <filename>/var/cache/homed/<replaceable>username</replaceable>/</filename>,
```

Comment:
hmm, I think this is systemd-specific enough to warrant to be placed inside of /var/cache/systemd/homed/, i.e. to put the "systemd" name in there.

usually, we only put super generic stuff we think makes sense for alternative implementsions too, outside of "systemd" dirs in /usr/lib/, /var or /run